ci-operator/step-registry/openshift-extended/test/disasterrecovery/openshift-extended-test-disasterrecovery-commands.sh

```diff
@@ -0,0 +1,565 @@
+    export SSH_CLOUD_PRIV_AZURE_USER="${QE_BASTION_SSH_USER:-core}"
+    export TEST_PROVIDER=azure
+    ;;
+azurestack)
+    export TEST_PROVIDER="none"
+    export AZURE_AUTH_LOCATION=${SHARED_DIR}/osServicePrincipal.json
+    ;;
+vsphere)
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/govc.sh"
+    export VSPHERE_CONF_FILE="${SHARED_DIR}/vsphere.conf"
+    error_code=0
+    oc -n openshift-config get cm/cloud-provider-config -o jsonpath='{.data.config}' > "$VSPHERE_CONF_FILE" || error_code=$?
+    if [ "W${error_code}W" == "W0W" ]; then
+        # The test suite requires a vSphere config file with explicit user and password fields.
+        sed -i "/secret-name \=/c user = \"${GOVC_USERNAME}\"" "$VSPHERE_CONF_FILE"
+        sed -i "/secret-namespace \=/c password = \"${GOVC_PASSWORD}\"" "$VSPHERE_CONF_FILE"
+    fi
+    export TEST_PROVIDER=vsphere;;
+openstack*)
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/cinder_credentials.sh"
+    export TEST_PROVIDER='{"type":"openstack"}';;
+ibmcloud)
+    export TEST_PROVIDER='{"type":"ibmcloud"}'
+    IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
+    export IC_API_KEY;;
+ovirt) export TEST_PROVIDER='{"type":"ovirt"}';;
+equinix-ocp-metal|equinix-ocp-metal-qe|powervs-*)
+    export TEST_PROVIDER='{"type":"skeleton"}';;
+nutanix|nutanix-qe|nutanix-qe-dis)
+    export TEST_PROVIDER='{"type":"nutanix"}';;
+*)
+    echo >&2 "Unsupported cluster type '${CLUSTER_TYPE}'"
+    if [ "W${FORCE_SUCCESS_EXIT}W" == "WnoW" ]; then
+        echo "do not force success exit"
+        exit 1
+    fi
+    echo "force success exit"
+    exit 0
+    ;;
+esac
+
+# create execution directory
+mkdir -p /tmp/output
+cd /tmp/output
+
+if [[ "${CLUSTER_TYPE}" == gcp ]]; then
+    pushd /tmp
+    curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-318.0.0-linux-x86_64.tar.gz
+    tar -xzf google-cloud-sdk-318.0.0-linux-x86_64.tar.gz
+    export PATH=$PATH:/tmp/google-cloud-sdk/bin
+    mkdir -p gcloudconfig
+    export CLOUDSDK_CONFIG=/tmp/gcloudconfig
+    gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+    gcloud config set project "${PROJECT}"
+    popd
+fi
+
+echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_START"
+
+# check if the cluster is ready
+oc version --client
+oc wait nodes --all --for=condition=Ready=true --timeout=15m
+oc wait clusteroperators --all --for=condition=Progressing=false --timeout=15m
+oc get clusterversion version -o yaml || true
+
+# execute the cases
+function run {
+    test_scenarios=""
+    hardcoded_filters="~NonUnifyCI&;~Flaky&;~DEPRECATED&;~SUPPLEMENTARY&;~VMonly&;~ProdrunOnly&;~StagerunOnly&"
+    echo "TEST_SCENARIOS_DISASTERRECOVERY: \"${TEST_SCENARIOS_DISASTERRECOVERY:-}\""
+    echo "TEST_ADDITIONAL_DISASTERRECOVERY: \"${TEST_ADDITIONAL_DISASTERRECOVERY:-}\""
+    echo "TEST_IMPORTANCE: \"${TEST_IMPORTANCE}\""
+    echo "TEST_FILTERS: \"${hardcoded_filters};${TEST_FILTERS:-}\""
```

Comment:
@geliu2016 
need to change TEST_FILTERS to TEST_FILTERS_DISASTERRECOVERY
please update it for the whole shell script.

Reply:
Updated.